v0.7/medical_imaging/3d-unet/Makefile

```diff
@@ -41,11 +42,18 @@ export nnUNet_raw_data_base=$(RAW_DATA_DIR)
 export nnUNet_preprocessed=$(PREPROCESSED_DATA_DIR)
 export RESULTS_FOLDER=$(RESULT_DIR)
 
-# Handle different nvidia-docker version
-ifneq ($(wildcard /usr/bin/nvidia-docker),)
-	DOCKER_RUN_CMD := nvidia-docker run
+HAS_GPU := $(shell command -v nvidia-smi 2> /dev/null)
+
+ifndef $HAS_GPU
+    DOCKER_RUN_CMD := docker run
 else
+
```

Comment:
Could you keep the comment `# Handle different nvidia-docker version`? Thanks

Reply:
Done.